features/support/env.rb

```diff
@@ -150,5 +150,15 @@ def extract_process_id(data)
   end
 
   expect(binaries.size).to eq(1)
-  expect(File.executable?(binaries[0])).to eq(true)
+  expect(File.executable?(binaries.first)).to eq(true)
+end
+
+Then(/^the binary "([^"]*)" should be installed with version "([^"]*)"$/) do |binary, version|
+  binaries = []
+  Find.find("#{ENV['VAGRANT_HOME']}/data/service-manager/bin") do |path|
```

Comment:
So here you are using the data directory to test something. Why can you not write a test which uses the --path options with a target of let's say  "#{ENV['VAGRANT_HOME']}/data/service-manager/custom-path". After completion the binary should be in the custom directory, right? BTW, if the custom directory does not exist, does it get created or what happens?